### PR TITLE
Update pagerduty integration docs

### PR DESCRIPTION
**Pull request by dtuite**

### CODE REVIEW

File: content/docs/integrations/pagerduty/index.md [hunk 1]
```diff
@@ -10,43 +10,100 @@ integrationType: OSS plugin
...
 ## Introduction
 
+The PagerDuty Backstage plugin allows Roadie Backstage users to:
+
+ 1. See who is on call for a particular catalog Component.
+ 2. See if there are active incidents for a given catalog Component.
+ 3. Create new incidents from Backstage.
+
+![PagerDuty plugin in Roadie Backstage](./pagerduty-plugin.png)
+
```
Comment (dtuite) [inline content/docs/integrations/pagerduty/index.md:20]: Added this information so the reader understands the value they will get from completing these steps.

File: content/docs/integrations/pagerduty/index.md [hunk 2]
```diff
@@ -10,43 +10,100 @@ integrationType: OSS plugin
...
 
 ```text
 https://<tenant-name>.roadie.so/administration/settings/secrets
 ```
 
-This page describes how to create and set up the API token.
+ℹ️  Roadie accepts the token from PagerDuty unmodified. The configuration steps for the PagerDuty plugin in self-hosted Backstage are more complex, and will not work in Roadie.
 
-## Steps
+### Step 3: Add the PagerDuty annotation
```
Comment (dtuite) [inline content/docs/integrations/pagerduty/index.md:55]: Added steps to show how to set the annotation.

**Review by punkle — APPROVED** [review]

File: content/docs/integrations/pagerduty/index.md (hunk 2 above)
Comment (punkle) [reply content/docs/integrations/pagerduty/index.md:55]: Looks great, thank you.

File: content/docs/integrations/pagerduty/index.md (hunk 1 above)
Comment (punkle) [reply content/docs/integrations/pagerduty/index.md:20]: 👍

**Pull request merged**